src/main/Validator.ts

```diff
@@ -92,6 +92,35 @@ const configDataSchemaV2 = Joi.object<ConfigV2>({
     downloadLocation: Joi.string(),
 });
 
+const configDataSchemaV3 = Joi.object<ConfigV3>({
+    version: Joi.number().min(2).default(2),
+    teams: Joi.array().items(Joi.object({
+        name: Joi.string().required(),
+        url: Joi.string().required(),
+        order: Joi.number().integer().min(0),
+        lastActiveTab: Joi.number().integer().min(0),
+        tabs: Joi.array().items(Joi.object({
+            name: Joi.string().required(),
+            order: Joi.number().integer().min(0),
+        })).default([]),
+    })).default([]),
+    showTrayIcon: Joi.boolean().default(false),
+    trayIconTheme: Joi.any().allow('').valid('light', 'dark').default('light'),
```

Comment:
```suggestion
    trayIconTheme: Joi.string().allow('').valid('light', 'dark').default('light'),
```

Reply:
Was `any` maybe used here to define a constant?

Reply:
looks like it, but why accept a number and then check that the only valid options `'' | 'light' | 'dark'` are actually strings?

Reply:
Idk, I didn't write this code, this is from the v2 schema :P